tareas/clase-6/clase6tarea1.js

```diff
@@ -0,0 +1,89 @@
+const $botonAgregarMiembros = document.querySelector(`#agregar-miembro`);
+const $botonCalcularEdad = document.querySelector(`#calcular-edad`);
+const $botonRemover = document.querySelector(`#remover`);
+
+$botonAgregarMiembros.onclick = function () {
+  const numeroMiembros = Number(
+    document.querySelector(`#miembros-familia`).value
+  );
+  if (numeroMiembros > 0) {
+    agregarMiembro(numeroMiembros);
+  } else {
+    alert("El valor 0 no es valido.");
+  }
+
+  return false;
+};
+
+function agregarMiembro(numeroMiembros) {
+  const $formulario = document.querySelector(`#formulario`);
+  for (i = 1; i <= numeroMiembros; i = i + 1) {
```

Comment:
There's missing the let keyword here.

``for (let i = 1; ...)``

Also, instead of using i = i + 1 you can use i++ if u want